src/router/router.tsx

```
@@ -1,6 +1,7 @@
 import EditProfile from "../Pages/EditProfile/EditProfile";
 import LocationVerifyPage from "../Pages/LocationVerifyPage/LocationVerifyPage";
 import Main from "../Pages/Main";
+import Map from "../Pages/MapPage/Map";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)